src/Components/LandingPage/LandingPage.tsx

```diff
@@ -53,7 +53,7 @@ export const LandingPage = () => {
     }
     setActiveTabKey(tabIndex);
   };
-  const [selectedBlueprint, setSelectedBlueprint] = useState<string>('');
+  const [selectedBlueprint, setSelectedBlueprint] = useState<string>();
```

Comment:
Would it make sense to initialize to `undefined`? I assume at some point we will need to be able to clear the filter, and I imagine we would reset to `undefined` in that case.

Reply:
it's already initialized to undefined. By default,  empty `useState` values are initiziled to undefined

Reply:
I'm kind of surprised we don't need to type the useState using `<string | undefined>`... does it infer that?